.github/workflows/ci.yml

```diff
@@ -0,0 +1,71 @@
+name: CI
+
+on:
+  # Run when main gets updated, i.e. PR merge
+  push:
+    branches: ["main"]
+
+  # Run when a PR is opened, and whenever new commits are added to the PR
+  pull_request:
+
+  # Run when manually triggered
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    permissions:
+      # Allow GitHub to request an OIDC JWT ID token, to use with aws-actions/configure-aws-credentials
+      id-token: write
+      contents: read
+
+    steps:
+      # Checkout the repository
+      - uses: actions/checkout@v4
+
+        # Get the desired version of Node installed
+      - uses: actions/setup-node@v3
+        with:
+          node-version-file: .nvmrc
+
+      # Execute some tasks from the `package.json` file, using `npm`
+      - name: CI
+        working-directory: cdk
+        run: |
+          npm ci
+          npm run postinstall
+          npm run bundlejs
+          npm run sass
+
+    # Get the desired version of Java installed (also comes with the SBT tool)
+    - uses: actions/setup-java@v3
```

Comment:
I think the reason the build is currently failing is because everything from here down needs to be indented so it's in line with the previous steps.

Reply:
yes, that was it! 
There was another problem after that - The npm CI step had a working directory ("cdk") step, but has to run from the project root. I've changed that and it seems okay now